src/poprox_recommender/training/train.py

```diff
@@ -144,11 +144,20 @@ def save_model(model):
     parser.add_argument("--num_words_title", type=float, default=30)
 
     args = parser.parse_args()
+
+    # set a seperate output directory for subset
+    if args.subset is not None:
+        args.output_dir = root / f"models/nrms-mind-subset-{args.subset}"
+    else:
+        args.output_dir = MODEL_DIR
```

Comment:
We should either just use an `output_dir` variable (don't put it on `args`), and pass it below (and ignore my comment below); or we should add a `--output-dir` argument in `add_argument` above, and only do this setting if `args.output_dir is None` (i.e., if they don't specify an output dir, we ovewrite it).

Reply:
The reason is that `args` should be the command line arguments. So if `--output-dir` is a command-line argument, then we should use `args.output_dir`; if it is not, then we should use a separate variable, to avoid surprising future readers.